lib/screens/v2/widget_instance/alert.ex

```diff
@@ -271,11 +272,12 @@ defmodule Screens.V2.WidgetInstance.Alert do
     end
   end
 
-  def audio_valid_candidate?(%__MODULE__{screen: %Screen{app_id: app_id}})
-      when app_id in ~w[bus_eink_v2 gl_eink_v2]a,
-      do: true
+  def audio_valid_candidate?(%__MODULE__{
+        screen: %Screen{app_params: %BusShelter{audio: %Audio{interval_enabled: true}}}
+      }),
+      do: false
 
-  def audio_valid_candidate?(_instance), do: false
+  def audio_valid_candidate?(_instance), do: true
```

Comment:
So does this mean now that any screen type that uses the Alert or Subway Status widgets will attempt an audio readout of them? Are there any implications of that we need to be concerned with or is that a non-issue?

Reply:
This should be logically equivalent — the existing screen types that use the Alert widget are Bus E-ink, GL E-ink, and Bus Shelter, and the screen types that use Subway Status are the same plus PreFare. In other words the logic was previously written as "enable audio on [list of all screen types that use this widget except for Bus Shelter], disable it otherwise". I thought this would be better expressed by inverting the condition, making "Bus Shelter with periodic audio" the special case where we _don't_ read out, and otherwise enable it.

Reply:
I see. Makes sense!